core/src/main/java/com/scalar/db/common/error/Category.java

```diff
@@ -0,0 +1,24 @@
+package com.scalar.db.common.error;
+
+import java.util.Objects;
+
+public enum Category {
```

Comment:
I think we should say something about retryability in the messages, the code, or the comments.
What do you think?

Reply:
I think we can add the retryability thing to the `solution` field later. What do you think?

Reply:
Ah, right. That should be OK.
I'm also wondering if we could have a reusable message (e.g., constant variable) for retryability because, for example, `CONCURRENCY_ERROR` is always retryable, so we should add the same description for retryability to all the concurrency errors' messages.
Maybe, we can think about it later.

Reply:
@feeblefakie 

> I'm also wondering if we could have a reusable message (e.g., constant variable) for retryability because, for example, CONCURRENCY_ERROR is always retryable, so we should add the same description for retryability to all the concurrency errors' messages.
> Maybe, we can think about it later.

Yeah, let's do that as the second step. In any case, we should think about adding the `cause` and `solution` field as the next step. I think we can do that with it. Thank you.